Resources/Help/analyses/ANCOVA.md

```diff
@@ -0,0 +1,227 @@
+# ANCOVA 
+---
+ANCOVA allows the user to analyze the difference among three, or more, group means, when partialling out the effect of variables that have an influence on the dependent variable but are not part of the experimental manipulation (covariates). 
+
+### Assumptions 
+- The dependent variable should be normally distributed for every group.
+- The independent variables are categorical, the dependent variable should be continuous or ordinal. 
+- The variance of the dependent variable should be the same for every group. This is called homogeneity. 
+- The groups are independent. 
+- The covariate and the experiment effect have to be independent. 
+- Homogeneity of the regression slopes, meaning that the effect of the covariate on the dependent variable is not allowed to differ between groups.  
+
+## Input 
+--- 
+### Assignment Box 
+- Dependent Variable: The variable you are interested in. Also called the outcome variable. 
+- Fixed Factors: The variables that you manipulated/define the different groups. Also called the independent variables.  
+- Covariates: In this box you can place the variable that is the covariate. Covariates are continuous variables that have an influence on the dependent variable but are not part of the experimental manipulation.  
+- WLS Weights: Weighted Least Squares, here you can place the variable specifying which points have more weight and are therefore considered more informative. For this last option it is important to know the weights a priori. This option is mainly used when the errors are heteroskedastic.    
+
+#### Model: 
+- Components and model terms 
+    - Components: All the independent variables and covariates you can put in the model. 
+    - Model terms: The independent variables and covariates being used in the model. By default all the independent variables and covariates included in the model are placed in this box, plus the interaction between the independent variables.  
```

Comment:
.. covariates included in the model. 
... By default all the main effects and interaction effects of the specified independent variables and the covariates are included in the model.